snuba/datasets/processors/errors_processor.py

```diff
@@ -327,6 +341,27 @@ def extract_stacktraces(
                     frame_linenos.append(_collapse_uint32(frame.get("lineno", None)))
                     frame_stack_levels.append(stack_level)
 
+                ## mark if at least one of the exceptions happened in the main thread
+                if thread_id is not None and exception_main_thread is not True:
```

Comment:
@marandaneto can we put this logic behind a snuba config option? 

if `get_config` is not already in this file you can import:  `from snuba.state import get_config` and then just set some flag with a default value so that we can turn this off without needing code changes if this ends up causing issues

```python
if get_config("check_exception_threads", 0):
   # your logic
```

Reply:
[`3186e1a` (#3863)](https://github.com/getsentry/snuba/pull/3863/commits/3186e1a3527c9de7b6a4f62f662b0f69654dc9f1)